src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+                         'install_nslcd',
+                         'misc_test',
+                         'netgroups')
+@pytest.mark.tier1_3
+class TestProxyMisc(object):
+    """
+    This is test case class for proxy provider suite
+    """
+    def test_bz1036758(self, multihost, backupsssdconf):
+        """
+        :title: Allow for custom attributes in RDN bz1036758
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1036758
+        :id: 10eb49a4-b252-11ec-87ea-845cf3eff344
+        :steps:
+          1. Add a local user
+          2. sssd to fetch id information from local user
+          3. Auth should work using ldap provider
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+        """
+        client_e = multihost.client[0].ip
+        tools = sssdTools(multihost.client[0])
+        tools.clear_sssd_cache()
+        # Auth should work using ldap provider
+        ssh1 = SSHClient(client_e, username="foo1", password="Secret123")
+        ssh1.close()
+        execute_cmd(multihost, "chown -R foo1 ~foo1")
+        execute_cmd(multihost, "chown -R foo1 /var/spool/mail/foo1")
+        # error logged in log
+        execute_cmd(multihost, "systemctl stop sssd.service")
+        execute_cmd(multihost, "rm -rf /var/lib/sss/{db,mc}/*")
+        execute_cmd(multihost, "systemctl start sssd.service")
+        execute_cmd(multihost, 'su - foo1 -c "logout"')
+
+    def test_bz785902(self, multihost):
+        """
+        :title: Errors with empty loginShell and proxy provider bz785902
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=785902
+        :id: 17831b7a-b252-11ec-9942-845cf3eff344
+        :steps:
+          1. Adding the user in ldap server with empty login shell
+          2. Search for Internal Error
+          3. Non existing netgroup returned with proxy
+          provider when proxy lib name is ldap
+        :expectedresults:
+          1. Should succeed
+          2. Should not succeed
+          3. Should not succeed
+        """
+        # Errors with empty loginShell and proxy provider bz785902
+        tools = sssdTools(multihost.client[0])
+        master_e = multihost.master[0].ip
+        ldap_uri = f'ldap://{master_e}'
+        ds_rootdn = 'cn=Directory Manager'
+        ds_rootpw = 'Secret123'
+        ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+        user_dn = 'uid=foo1,ou=People,dc=example,dc=test'
+        del_member = [(ldap.MOD_REPLACE, 'loginShell', "".encode('utf-8'))]
+        (ret, _) = ldap_inst.modify_ldap(user_dn, del_member)
+        assert ret == 'Success'
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'ldap',
+                         'auth_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
```

Comment:
See my other comment about having the different SSSD configurations in individual fixtures.

Reply:
done